### DISCUSSION TITLE
What if somebody wanted to give you money nevertheless?

### DISCUSSION THREAD

**Discussion by larss** [redphx/better-xcloud#126] [General]
Ok, you don't want donations, but your work is remarkable and is allowing me to play xcloud (and Starfield by instance) as good as it should be played.
I reckon that at least a couple of beers are due. Is there a Paypal available somewhere?

larss

**Comment by redphx** [comment]
I have setup a ko-fi page at https://ko-fi.com/redphx  
Thank you.

**Reply by larss** [reply]
Thanks to you!
